src/testdrive/src/action.rs

```diff
@@ -553,7 +556,10 @@ fn substitute_vars(msg: &str, vars: &HashMap<String, String>) -> Result<String,
     let mut err = None;
     let out = RE.replace_all(msg, |caps: &Captures| {
         let name = &caps[1];
-        if let Some(val) = vars.get(name) {
+        if name == INGEST_ITERATION_RUNTIME_VAR {
+            // This variable is substituted at $ kafka-ingest time
+            &INGEST_ITERATION_RUNTIME_VAR_DECORATED
```

Comment:
There is no way in Rust that I could find to opt out of performing a particular replacement. At the same time, the lifetime of the thing you are replacing with needs to outlive the lambda, so the only way I could do it is by defining it as a `const` str.

Reply:
I think capture group 0 is the full match, so I think you can probably just return `caps.get(0)` in this case?